tests/BladeDirectivesTest.php

```diff
@@ -90,6 +90,29 @@ public function it_not_displays_impersonating_content_directive()
         $this->logout();
     }
 
+    /** @test */
+    public function it_displays_not_impersonating_content_directive() {
+        $this->actingAs($this->user);
+        $this->makeView();
+        $this->assertStringContainsString('Not impersonating this user', $this->view);
+        $this->logout();
+
+        $this->actingAs($this->admin);
+        $this->makeView();
+        $this->assertStringContainsString('Not impersonating this user', $this->view);
+        $this->logout();
+    }
+
+    /** @test */
+    public function it_not_displays_not_impersonating_content_directive() {
+        $this->actingAs($this->admin);
+        $this->admin->impersonate($this->user);
+        $this->makeView();
+        $this->assertStringNotContainsString('Not impersonating this user', $this->view);
+        $this->admin->leaveImpersonation();
+        $this->logout();
+    }
```

Comment:
The other tests don't call ->impersonate and ->leaveImpersonation, so I wonder if it would make sense to add another assertion to ensure that after leaving impersonation the Not Impersonating message re-appears?